.github/workflows/verify-dashboards-alerts.yaml

```diff
@@ -0,0 +1,385 @@
+
+          echo "ACCESS_TOKEN=$(echo $RESPONSE | jq -r .token)" >> $GITHUB_ENV
+
+      - name: Generate Registration Token
+        run: |
+          REPO_OWNER=${{ github.repository_owner }}
+          REPO_NAME=${{ github.event.repository.name }}
+          RESPONSE=$(curl -L \
+            -X POST \
+            -H "Authorization: Bearer ${{ env.ACCESS_TOKEN }}" \
+            https://api.github.com/repos/$REPO_OWNER/$REPO_NAME/actions/runners/registration-token)
+
+          echo "REGISTRATION_TOKEN=$(echo $RESPONSE | jq -r .token)" >> $GITHUB_ENV
+
+      - name: SSH into EC2 Instance and Register Runner
+        env:
+          REGISTRATION_TOKEN: ${{ env.REGISTRATION_TOKEN }}
+        run: |
+          sudo apt-get update -y
+          sudo apt-get install -y libicu-dev
+
+          # Function to check SSH connectivity
+          function check_ssh {
+              nc -z -w 5 ${{ env.EC2_PUBLIC_IP }} 22
+          }
+
+          echo "Checking SSH connectivity to ${{ env.EC2_PUBLIC_IP }}..."
+
+          # Loop until SSH is available
+          while ! check_ssh; do
+              echo "SSH is not yet available. Retrying in 10 seconds..."
+              sleep 10
+          done
+
+          echo "SSH is now available. Connecting now..."
+
+          ssh -o StrictHostKeyChecking=no -i ${{ secrets.AWS_KEY_NAME }}.pem ubuntu@${{ env.EC2_PUBLIC_IP }} << EOF
+            mkdir actions-runner && cd actions-runner
+            curl -o actions-runner-linux-x64-2.317.0.tar.gz -L https://github.com/actions/runner/releases/download/v2.317.0/actions-runner-linux-x64-2.317.0.tar.gz
+            tar xzf ./actions-runner-linux-x64-2.317.0.tar.gz
+
+            ./config.sh --url https://github.com/${{ github.repository_owner }}/${{ github.event.repository.name }} --token ${{ env.REGISTRATION_TOKEN }} --unattended --labels self-hosted,linux,aws
+            nohup ./run.sh > nohup.out 2> nohup.err < /dev/null &
+          EOF
+
+  verify-dashboards-alerts:
+    name: Verify Dashboards and Alerts OK
+    needs: deploy-register-self-runner
+    runs-on: self-hosted
+    defaults: 
+      run:
+        shell: bash -ieo pipefail {0}
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Run Quickstart
+        run: |
+          sudo chown $(whoami) .
+          kind delete clusters kuadrant-local
+          kind delete clusters kuadrant-local
+
+          export ISTIO_INSTALL_SAIL=true
+
+          cat << 'EOF' > automate_setup.expect
+          #!/usr/bin/expect -f
+          set timeout -1
+          spawn bash hack/quickstart-setup.sh
+          expect "Are you ready to begin? (y/n)"
+          send "y\r"
+          expect "Do you want to set up a DNS provider for use with Kuadrant's DNSPolicy API? (y/n)"
+          send "n\r"
+          expect "Thank you for using Kuadrant! If you have any questions or feedback, please reach out to our community."
+          EOF
```

Comment:
Being able to run the quickstart script to completion is a very useful thing, outside of this specific workflow.
I'm not suggesting any changes at this time, but I'm wondering if you have given any thought on how this might be reused for setting up a CI test environment for use by different acceptance tests (like these dashboard & alerts tests). I'd imagine an acceptance test around a gateway and policies would be very useful to have in a workflow too.

Reply:
It did cross my mind as to how it could be a universal way to run quickstart. I didn't implement it that way for this PR as I didn't know how such a huge PR would be perceived.  :D 

I think it definitely makes sense to have an implementation that can be called for multiple CI/CD pipelines. :)

Reply:
My suggestion would be to add `quiet` flag feature to quickstart itself and the ability to specify everything through CLI args. Of course, this would be different issue and PR, but I think it would be worth it to implement that

Reply:
@pehala I think this makes sense. Would you prefer it this PR waited for those changes ? Therefore, I could implement it directly ?

Reply:
https://github.com/Kuadrant/kuadrant-operator/pull/708#discussion_r1668422907

Reply:
Marking as resolved as new changes make waiting for quickstart changes redundant.

Reply:
https://github.com/Kuadrant/kuadrant-operator/issues/743